samples/nrf9160/modem_shell/src/rest/rest_shell.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <zephyr.h>
+#include <assert.h>
+
+#include <shell/shell.h>
+#include <getopt.h>
+
+#include <net/http_parser.h>
+#include <net/rest_client.h>
+
+#include "rest_shell.h"
+#include "mosh_print.h"
+
+static const char rest_shell_cmd_usage_str[] =
+	"Simple REST shell client\n"
```

Comment:
Maybe removing word "simple" from here. It's not wrong and, e.g., in documentation I guess it could still be used in few places but I'd probably remove it from the descriptions in the code.

Reply:
You really don't like it as simple ;-)
OK, I can remove it but this really is a SIMPLE REST client.

Reply:
Yep, but I still left it in the documentation :)

Reply:
yeah, and stays until it is something else than simple ;-)

Reply:
That's fine. When you make this more advanced, there is just one "simple" word in the documentation to be removed :)